crowbar_framework/app/models/backup.rb

```diff
@@ -114,10 +117,10 @@ def create_archive
   end
 
   def save_archive
-    self.name = file.original_filename.split(".").first
+    self.name = file.original_filename.remove(".tar.gz")
```

Comment:
we have that in the frontend at least, this is taken care of by the fileinput.js